tasks/Splitting a deck of cards.md

```diff
@@ -0,0 +1,59 @@
+Laughlin and Shupe (1996) allowed for 32 different rules (or patterns), which contained four patterns that were of length five.  This is different from other experiment spaces.  There description of the rules is as follows: 
+> The correct rules were patterns of alternation over four or five cards (e.g., ‘‘two diamonds alternate with two clubs’’). Abbreviating clubs as C, diamonds as D, hearts as H, spades as S, red (diamonds or hearts) as R, and black (clubs or spades) as B, the 32 rules were DDCC, DDHH, DDSS, SSDD, SSCC, SSHH, DCHS, DHCS, DHSC, SCHD, SCDH, SHCD, DCCC, DHHH, DSSS, SDDD, SCCC, SHHH, DDDS, SSSD, DDDDS, SSSSD, RRBB, BBRR, RBBB, BRRR, RRRB, BBBR, RRBBB, BBRRR, RRRBB, and BBBRR, with one rule for each of the 32 replications. (Laughlin and Shupe, 1996)
+
+
+Other versions of the experiment, such as McGlynn, et al.  allowed for a brain storming session on either trial 3 (third card drawn) or trial 9 (ninth card drawn).  During these brain storming sessions, groups/individuals were given the following instructions.
+> * (1)  Criticism is ruled out. Make sure each hypothesis fits the cards, but otherwise adverse judgment of ideas must be withheld (and no one should criticize any- one elseÕs ideas/so donÕt criticize any ideas that come to mind). Say everything you think of that fits the cards.  
+> * (2)  Freewheeling is welcome. The more creative the idea, the better. It is easier to tame down than to think up. DonÕt be afraid to say anything that comes to mind. As long as it fits the cards, the more un- usual the hypothesis the better. This will stimulate more and better ideas.  
+> * (3)  Combination and improvement are sought. (You should try to suggest how ideas of others can be joined or changed into still better ideas./You should try to join or change ideas you have into still better ideas). DonÕt be afraid to combine and improve on them.  
+> * (4)  Most important: quantity is wanted. The greater the number of ideas, the more the likelihood of winners. Come up with as many as you can that fit the cards 
+The experimenter would read these rules to participants and then instructed them to brainstorm for 10 minutes.  The instructor would then start a tape recorder and leave the room.  The experimenter would enter the room at the 3, 6 and 9 minute mark, and if no brainstorming was happening, they would prompt the group as follows; “Keep going. Please don’t forget that it’s the total number of ideas that counts in brainstorming.”  At the 10 minute mark the experimenter would enter the room, stop the tape recorder and instruct the participants to continue the experiment, starting by writing down individual hypotheses for the third or ninth card.
+
+# Procedure
+## Steps
+1. Participants are randomly broken into groups of four (or five in the case of Laughlin and Shipply) or are selected to complete the task in the nominal(individual) setting.
+2. Participants then followed instructions in an attempt to find the rule.
```

Comment:
I would add a bit more detail here: 
 2. Members individually record a single hypothesis about what the correct rule might be. 
 3. Then the group engages in discussion to consensus on a group hypothesis.
 4.  Finally, the group tests the hypothesis by selecting another card in order to learn if it is an example or not. This is analogous to generating and testing a scientific hypothesis. 
 5. Groups receive feedback on the status each card selected (i.e., the results of the “experiment”) but no feedback on the hypothesis itself
 6. On each trial, groups can see all the evidence (cards) that has accumulated from previous trials.
 7.  The cycle of member hypothesis, discussion, group hypothesis, card selection, and feedback continues for 10–15 trials